sphinx_gallery/gen_gallery.py

```diff
@@ -1503,8 +1503,9 @@ def setup(app):
     # Early update of sphinx_gallery_conf at builder-inited
     app.connect("builder-inited", update_gallery_conf_builder_inited, priority=10)
     app.connect("builder-inited", generate_gallery_rst)
-    app.connect("build-finished", copy_binder_files)
-    app.connect("build-finished", create_jupyterlite_contents)
+    # Setting a small priority so that create_jupyterlite_contents runs before
```

Comment:
```suggestion
    # Setting a low priority so that create_jupyterlite_contents runs before
```

Just to avoid confusion.

Reply:
So I used small here on purpose rather than low. Callbacks are run in order of ascending priority so small priority means high priority (`priority=100` means higher priority than `priority=500` which is the default). 

Maybe "setting priority to a small number" is less confusing. There is a similar in a comment a few lines above so it could be improved as well ...

Reply:
Ah I got myself confused, even after reading the docs.

"setting priority to a small number" sounds good!

Reply:
OK I tweaked the comments hopefully avoiding the confusion a bit more.

For reference, the `connect` [doc](https://www.sphinx-doc.org/en/master/extdev/appapi.html#sphinx.application.Sphinx.connect) says:

> priority: The priority of the callback. The callbacks will be invoked in order of priority (ascending).